src/components/editor/DraftSpec.tsx

```diff
@@ -26,19 +24,18 @@ function DraftSpecEditor() {
     const [draftSpec, setDraftSpec] = useState<DraftSpecQuery | null>(null);
 
     const handlers = {
-        change: (newVal: any) => {
+        change: (newVal: any, catalogName: string) => {
```

Comment:
Since there is not an existing variable named `value`, prefixing the first argument with `new` seems odd since a comparison is not performed.

As for the second argument, is the proper term for that segment of data _catalog name_ or _catalog namespace_? Even if both terms are suitable, we should select one and use that throughout the codebase to prevent viewers/contributors from believing there is a distinction between instances of `catalogNamespace` and `catalogName`. Whichever term will appear in Estuary's documentation and used in communication should be the term we move forward with in the codebase.

Reply:
Since sometimes we need new names for variables inside functions due to scoping issues I try to keep them consistent. Hence `newVal`.

Yeah I agree but we can worry about that when we're done.